hudi-examples/src/main/java/org/apache/hudi/examples/deltastreamer/HoodieDeltaStreamerKafkaSourceExample.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.examples.deltastreamer;
+
+import org.apache.hudi.DataSourceWriteOptions;
+import org.apache.hudi.examples.common.HoodieExampleDataGenerator;
+import org.apache.hudi.examples.common.HoodieExampleSparkUtils;
+import org.apache.hudi.utilities.deltastreamer.HoodieDeltaStreamer;
+import org.apache.hudi.utilities.sources.JsonKafkaSource;
+import org.apache.hudi.utilities.transform.IdentityTransformer;
+
+import com.beust.jcommander.JCommander;
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaSparkContext;
+
+
+/**
+ * Simple examples of #{@link HoodieDeltaStreamer} from #{@link JsonKafkaSource}.
+ *
+ * To run this example, you should
+ *    1. Start Zookeeper and the Kafka demo server
+ *    2. For running in IDE, set VM options `-Dspark.master=local[2]`
+ *    3. For running in shell, using `spark-submit`
+ *    4. produce some data to hoodie-source-topic configured by `hoodie.deltastreamer.source.kafka.topic`
+ *
+ * Usage: HoodieDeltaStreamerKafkaSourceExample \
+ *        --target-base-path /tmp/hoodie/kafkadeltatable \
+ *        --table-type MERGE_ON_READ \
+ *        --target-table kafkadeltatable
+ */
+public class HoodieDeltaStreamerKafkaSourceExample {
+
+  public static void main(String[] args) throws Exception {
+
+    final HoodieDeltaStreamer.Config cfg = defaultKafkaDeltaStreamerConfig();
+    new JCommander(cfg).parse(args);
+
+    SparkConf sparkConf = HoodieExampleSparkUtils.defaultSparkConf("hoodie-delta-streamer-kafka-source-example");
+    JavaSparkContext jsc = new JavaSparkContext(sparkConf);
+
+    try {
+      new HoodieDeltaStreamer(cfg, jsc).sync();
+    } finally {
+      jsc.stop();
+    }
+  }
+
+  /**
+   * also see #{@link HoodieDeltaStreamer.Config} for more params.
+   * @return default params for Kafka DeltaStreamer
+   */
+  private static HoodieDeltaStreamer.Config defaultKafkaDeltaStreamerConfig() {
+
+    HoodieDeltaStreamer.Config cfg = new HoodieDeltaStreamer.Config();
+
+    cfg.configs.add(String.format("%s=uuid", DataSourceWriteOptions.RECORDKEY_FIELD_OPT_KEY()));
```

Comment:
little more comments for important configs to guide the user? for e.g 

for this , 
`    cfg.configs.add("bootstrap.servers=localhost:9092");`

we can `// The kafka cluster we want to ingest from`

Reply:
ok, good.